recipes/r-indicspecies/meta.yaml

```
@@ -15,10 +15,10 @@ source:
 build:
   merge_build_host: True  # [win]
   number: 0
-  noarch: generic
+  skip: True  # [win]
```

Please revert. This is a `noarch` package and should remain so, and we ignore - rather than skip - non-**linux-64** builds in such cases:

```suggestion
  noarch: generic
```